module/config.mjs

```diff
@@ -2447,6 +2451,29 @@ preLocalize("distanceUnits");
 
 /* -------------------------------------------- */
 
+/**
+ * The valid units for measurement of volume.
+ * @enum {UnitConfiguration}
+ */
+DND5E.volumeUnits = {
+  cubicFoot: {
+    label: "DND5E.UNITS.VOLUME.CubicFoot.Label",
+    abbreviation: "DND5E.UNITS.Volume.CubicFoot.Abbreviation",
+    counted: "DND5E.UNITS.Volume.CubicFoot.Counted",
+    conversion: 1,
+    type: "imperial"
+  },
+  liter: {
+    label: "DND5E.UNITS.VOLUME.Liter.Label",
+    abbreviation: "DND5E.UNITS.Volume.Liter.Abbreviation",
+    conversion: 1 / 28.317, // Should we do an approximate conversion here? Does it matter?
```

Comment:
I'm sure localized 5e books have some simplified conversion of cubic feet to liters. Got an example?